File: Beecrowd/1073 Quadrado de Pares/main.cpp
#include <iostream>
#include <math.h>
using namespace std;
 
int main() {
 
	int i, N, X;
	
	do{

	cin >> N;
	}while(N < 5 || N > 2000);
	
	for(i = 1; i <= N; i++){
		if(i % 2 == 0){
			X = pow(i, 2);
			cout << i << "^2 = " << X << endl;	
		}
	}
 
    return 0;
}

File: Beecrowd/1073 Quadrado de Pares/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws Exception {
        Scanner sc = new Scanner(System.in);

        int n, x;

        n = sc.nextInt();

        for (int i = 1; i <= n; i++) {
            if (i % 2 == 0) {
                x = (int) Math.pow(i, 2.0);
                System.out.println(i + "^2 = " + x);
            }
        }

        sc.close();
    }
}